modules/exploits/unix/webapp/spip_rce_form.rb

```diff
@@ -0,0 +1,124 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'SPIP form PHP Injection',
+        'Description' => %q{
+          This module exploits a PHP code injection in SPIP. The vulnerability exists in the
+          oubli parameter and allows an unauthenticated user to execute arbitrary commands
+          with web user privileges. Branches 3.2, 4.0, 4.1 and 4.2 are concerned. Vulnerable versions
+          are <3.2.18, <4.0.10, <4.1.18 and <4.2.1.
+        },
+        'Author' => [
+          'anonymous', # Initial discovery
+          'Laluka',         # PoC
+          'Julien Voisin'   # MSF module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'URL', 'https://blog.spip.net/Mise-a-jour-critique-de-securite-sortie-de-SPIP-4-2-1-SPIP-4-1-8-SPIP-4-0-10-et.html' ],
+        ],
+        'Privileged' => false,
+        'Platform' => ['php'],
+        'Arch' => ARCH_PHP,
+        'Targets' => [
+          [ 'Automatic', {} ],
+        ],
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
+          'Reliability' => [ REPEATABLE_SESSION ],
+          'SideEffects' => []
+        },
+        'Payload' => {
+          'BadChars' => "\x22\x00"
+        },
+        'DefaultTarget' => 0,
+        'DisclosureDate' => '2023-02-27'
```

Comment:
Is this meant to be Monday January 16th, 2023 ?

2023-02-27 looks like it's associated with this CVE / Blog Post which seems to be a different exploit

- https://nvd.nist.gov/vuln/detail/CVE-2023-24258
- https://blog-spip-net.translate.goog/Mise-a-jour-critique-de-securite-sortie-de-SPIP-4-2-1-SPIP-4-1-8-SPIP-4-0-10-et.html?lang=fr&_x_tr_sl=ja&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp

Reply:
https://blog.spip.net/Mise-a-jour-critique-de-securite-sortie-de-SPIP-4-2-1-SPIP-4-1-8-SPIP-4-0-10-et.html?lang=fr nope, it's 27th of March 2023.

Reply:
I think `2023-02-27` is correct. Which just for clarity's sake was posted on the 27th of **February** 2023.

@adfoster-r7, the CVE you linked [CVE-2023-24258](https://nvd.nist.gov/vuln/detail/CVE-2023-24258) seems to be an SQLi in the `_oups`. Which might be different as this seems to be a command injection via the `oubli` post parameter. 

I think this exploit might be for: https://nvd.nist.gov/vuln/detail/CVE-2023-27372, in which case we could add this CVE.  The reason I'm not certain is this link which I assume would have been the PoC is now a 502: https://therealcoiffeur.com/c11010